core/commands/pin.go

```diff
@@ -455,14 +480,14 @@ type RefKeyList struct {
 	Keys map[string]RefKeyObject
 }
 
-func pinLsKeys(ctx context.Context, args []string, typeStr string, n *core.IpfsNode, api iface.CoreAPI) (map[string]RefKeyObject, error) {
+func pinLsKeys(ctx context.Context, args []string, typeStr string, n *core.IpfsNode, api iface.CoreAPI) (map[cid.Cid]RefKeyObject, error) {
```

Comment:
This logic is unfortunately still duplicated in `core/coreapi/pin.go`, do you mind updating it there too? (this will get fixed once pin commands are rewritten to use CoreAPI)

Reply:
I can do that in a separate P.R.

Reply:
Done see: https://github.com/ipfs/go-ipfs/pull/5809